cernopendata_client/cli.py

```diff
@@ -93,6 +85,11 @@ def get_record(recid, doi, title, output_fields):
               help='Expand file indexes? [default=yes]')
 def get_file_locations(recid, doi, title, protocol, expand):
     """Get a list of files belonging to a dataset."""
+    if recid is not None and recid <= 0:
```

Comment:
This validation is coming two times, Should I go ahead with creating a sepearte function and calling it or this is fine?

Reply:
Yes, we can refactor everything... There will be many more functions in the future, so it will be useful to take the validation out. 

Speaking pseudo-code style, we currently have three ways how to select records: via recid, via doi, via title.  These can maybe later accept wildcards so that the user can comfortably select more than one related dataset (e.g. `title='/BTag/*'`). Imagine we have `get-record` now, but we add a new `get-records` in the future. So why don't we introduce a "validate" function that we could plug everywhere where it is needed, but also another "select" function that will return list of record IDs based by matching recid, or doi, or title, and the caller can decide what to do with the findings, e.g. if somebody asks for one record in `get-record` we first run the validator function to make sure arguments are OK, then the record selector function to find matching record IDs, and if the selector finds more than one record, then we should abort with an error, and if it finds one record, we get JSON for it by hitting REST API.